ts/modules/draggable-points.src.ts

```diff
@@ -0,0 +1,3045 @@
+                validateIndividualDrag: function (
+                    point: Highcharts.GanttPoint
+                ): boolean {
+                    return !point.milestone;
+                }
+            }),
+            /**
+             * Allow end value to be dragged individually. Requires
+             * `draggable-points` module.
+             *
+             * @type      {boolean}
+             * @default   true
+             * @apioption plotOptions.gantt.dragDrop.draggableEnd
+             */
+            end: merge(xrangeDragDropProps.x2, {
+                optionName: 'draggableEnd',
+                // Do not allow individual drag handles for milestones
+                validateIndividualDrag: function (
+                    point: Highcharts.GanttPoint
+                ): boolean {
+                    return !point.milestone;
+                }
+            })
+        };
+    }
+}
+
+// Don't support certain series types
+[
+    'gauge',
+    'pie',
+    'sunburst',
+    'wordcloud',
+    'sankey',
+    'histogram',
+    'pareto',
+    'vector',
+    'windbarb',
+    'treemap',
+    'bellcurve',
+    'sma',
+    'map',
+    'mapline'
+].forEach(
+    /**
+     * @private
+     * @param {string} type
+     *        Unsupported series type
+     * @return {void}
+     */
+    function (type: string): void {
+        if (seriesTypes[type]) {
+            seriesTypes[type].prototype.dragDropProps = null;
+        }
+    }
+);
+
+/**
+ * The draggable-points module allows points to be moved around or modified in
+ * the chart. In addition to the options mentioned under the `dragDrop` API
+ * structure, the module fires three events,
+ * [point.dragStart](plotOptions.series.point.events.dragStart),
+ * [point.drag](plotOptions.series.point.events.drag) and
+ * [point.drop](plotOptions.series.point.events.drop).
+ *
+ * It requires the `modules/draggable-points.js` file to be loaded.
+ *
+ * @sample highcharts/dragdrop/resize-column
+ *         Draggable column and line series
+ * @sample highcharts/dragdrop/bar-series
+ *         Draggable bar
+ * @sample highcharts/dragdrop/drag-bubble
+ *         Draggable bubbles
+ * @sample highcharts/dragdrop/drag-xrange
+ *         Draggable X range series
+ *
+ * @since     6.2.0
+ * @apioption plotOptions.series.dragDrop
+ */
+
+/**
+ * The amount of pixels to drag the pointer before it counts as a drag
+ * operation. This prevents drag/drop to fire when just clicking or selecting
+ * points.
+ *
+ * @type      {number}
+ * @default   2
+ * @since     6.2.0
+ * @apioption plotOptions.series.dragDrop.dragSensitivity
+ *
+ * @private
+ */
+var defaultDragSensitivity = 2;
+
+/**
+ * Style options for the guide box. The guide box has one state by default, the
+ * `default` state.
+ *
+ * @type         {Highcharts.Dictionary<Highcharts.PlotSeriesDragDropGuideBoxDefaultOptions>}
+ * @since        6.2.0
+ * @optionparent plotOptions.series.dragDrop.guideBox
+ *
+ * @private
```

Comment:
These options should go in the API - does the `@private` prevent this?

Reply:
The `@private` prevents them to go into the global namespace of the class-reference. `@ignore-option` is the equivalent for the API option tree.